index.js

```diff
@@ -12,13 +8,18 @@ require('./lib/objection')
 /**
  * GraphQL related code
  */
-const cookie = require('cookie')
-const jwt = require('jsonwebtoken')
+const { Action } = require('./models/Action')
 const { ApolloServer } = require('apollo-server')
 const { GraphQLJSONObject } = require('graphql-type-json')
+const { Model } = require('objection')
+const { queryResolvers, mutationResolvers, setContext } = require('./resolvers')
+const Knex = require('knex')
+const knexConfig = require('./knexfile.js')
 const typeDefs = require('./schema')
-const { Action } = require('./models/Action')
-const { queryResolvers, mutationResolvers } = require('./resolvers')
+
+// Initialize knex.
```

Comment:
@duranmla This should be how this PR has it https://github.com/debtcollective/campaign-api/pull/13/files#diff-168726dbe96b3ce427e7fedce31bb0bcR10

Reply:
Thanks a lot for this. The constant rebase is causing issues for me.